pdpyras.py

```diff
@@ -11,151 +11,732 @@
 from random import random
 
 import requests
+from logging import StreamHandler
 from urllib3.exceptions import HTTPError, PoolError
 from requests.exceptions import RequestException
 
 __version__ = '4.5.2'
```

Comment:
This might be because it's a WIP still, but I noticed the version string.

Reply:
better to update early than forget to do it later, thanks